9/Dockerfile

```diff
@@ -63,11 +64,26 @@ RUN mkdir -p /var/run/redis && \
     greenbone-nvt-sync && \
     greenbone-scapdata-sync && \
     greenbone-certdata-sync && \
-    BUILD=true /start && \
+    BUILD=true /start && \ 
     service openvas-scanner stop && \
     service openvas-manager stop && \
     service openvas-gsa stop && \
     service redis-server stop
 
+RUN apt-get update && \
+    apt-get install -y postfix supervisor libsasl2-modules syslog-ng-core mailutils && \
+    apt-get autoremove -yq
+
+COPY sasl_passwd_template /
+COPY main.cf_template /
+
+ENV BUILD=""
+
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
```

Comment:
We can separate it out if that is what you want. The issue is the container does not exit correctly as it is currently. So instead processes are being killed before they can exit gracefully. 

We can are happy to submit a separate PR if that is preferable